Car.cpp

```diff
@@ -1,7 +1,27 @@
 #include "Car.hpp"
 #include <iostream>
 
-void Car::turnLeft()      { std::cout << __FUNCTION__ << std::endl; }
-void Car::turnRight()     { std::cout << __FUNCTION__ << std::endl; }
-void Car::brake()         { std::cout << __FUNCTION__ << std::endl; }
-void Car::accelerate(int) { std::cout << __FUNCTION__ << std::endl; }
+void Car::turnLeft(){ 
+    direction_ = "left";
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight(){ 
+    direction_ = "right"; 
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake(){ 
+    speed_ = 0;
+    direction_ = "straight";
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::accelerate(int speed){
+    if (speed >= 0 && speed < maxSpeed_){
+        speed_ += speed;
```

Comment:
this is wrong. speed_ could have a value 20. And max speed value 100. What happens when you provide a speed argument as 90? It will pass if statement. and you got 110 speed which is above max limit.